pandas/tests/series/indexing/test_where.py

```diff
@@ -464,3 +464,34 @@ def test_where_categorical(klass):
     df = klass(["A", "A", "B", "B", "C"], dtype="category")
     res = df.where(df != "C")
     tm.assert_equal(exp, res)
+
+
+@pytest.mark.parametrize("tz", [None, "US/Pacific"])
```

Comment:
could use the timezone fixtures, but nbd

Reply:
?

Reply:
sure